etc/kayobe/inventory/group_vars/compute/firewall

```diff
@@ -47,6 +47,11 @@ firewallgen_ipv4_input_allow_rules:
     proto: tcp
     destination: "{{ internal_net_name | net_ip }}"
     comment: "hint: used by 'mtail' in docker container 'prometheus_mtail'"
+  - interface: "{{ internal_net_name | net_interface }}"
+    port: 10000:20000
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "Nova serial console proxy"
   - interface: "{{ internal_net_name | net_interface }}"
     port: 18080
     proto: tcp
```

Comment:
See line 66 below, I think its still needed because it goes between containers, but its a shame its not on box only for the libvirt port range.

Reply:
So think these server 2 different purposes; 5900:6000 is getting to the actual proxied serial console, 10000:20000 is for the console logfile. Example from a running instance domain XML:

    <graphics type='vnc' port='5900' autoport='yes' listen='10.205.1.34'>
      <listen type='address' address='10.205.1.34'/>
    </graphics>

    <console type='tcp'>
      <source mode='bind' host='10.205.1.34' service='10006' tls='no'/>
      <protocol type='raw'/>
      <log file='/var/lib/nova/instances/d31bef70-1fe0-47ed-83d1-a0f14aaa6f39/console.log' append='off'/>
      <target type='serial' port='0'/>
      <alias name='serial0'/>
    </console>